docs/system-overview.md

```diff
@@ -0,0 +1,77 @@
+To let the simulation know what is expected from it, the simulator will send mission controll messages a few seconds before the event start.
+These messages contain information about the mission (trackdrive, autocross, etc).
+When the autonomous system receives a mission message it can expect to receive sensordata shortly after.
+Since the laptimer for all events start whenever the car crosses the start-line, the autonomous system's can take all the time they need to launch their relevant algorithms for the mission (within reasonable time).
+
+## The Simulator
+
+The simulator is a continuously running program that is like the spider in the web.
+
+The simulator it offers a webinterface to the event's officials. 
+Using this webinterface the officials can choose what team/car is going to drive and which track is going to be used.
+The official will also be able to send the start signal, view lap times, down or out cones, car off course's.
+There is also an emergency stop button in case the car is uncontrollable.
+
+The simulator connects to the simulated world inside Unreal Engine using AirSim (more on that later).
+It also connects to the autonomous systems and acts like a bridge between the two.
+On one hand it requests sensordata and passes it along on ros topics to the current autonomous system.
+On the other hand it receives car controll commands from the autonomous system and forwards it to the virtual world.
+
+Only 1 car will be able to run on this autonomous system.
+The official can select what team, and thus what autonomous system, is currently selected.
+When the team changes not only the autonomous system, but also the car inside the virtual world changes.
+The sensor suite (sensor types and locations) and the car livery (looks of the car) are updated.
+The simulator keeps track of these details and passes them along to the virtual world to ensure acurate representation.
+
+When a start signal is received the simulator informs the current autonomous system of the mission and starts sending sensordata it's way.
+When a stop signal is received, the simulator stops interacting with the attached autonomous system and informs the virtual world that the car must stop driving
+
+During a mission the simulator keeps polling the world for 'referee state'. 
+This is information that in a phisical world would be relevant to the referee.
+Currently this includes a list of down or out cones and the timestamp of when they went down or out, a list of laptimes and a list of when the car went off-track.
+More information will for sure be added.
+
+In a single logbook is stored what happend inside the simulation.
+This includes all referee updates, what autonomous systems were selected and what tracks were used.
+If something unexpectedly went wrong like a system's crash or error, a short description of what happend is included in this logbook.
+It gives a timeline of everything that happend to always go back afterwards and check what happend.
```

Comment:
It gives a timeline of everything that happend, so that one can always check it afterwards.

Something about this sentence feels off, but I'm not sure how to rephrase it.